llvm/unittests/ADT/StringRefTest.cpp

```diff
@@ -47,37 +34,39 @@ static_assert(std::is_assignable_v<StringRef &, const char *&&>,
 static_assert(std::is_assignable_v<StringRef &, const char *&>,
               "Assigning from lvalue C string");
 
-namespace {
-TEST(StringRefTest, Construction) {
+// Test fixture.
+class StringRefTest : public testing::Test {};
```

Comment:
Why do you need it with no methods?

Reply:
Because that will come when I add test cases for reverse iterators, in a follow-on change. I assume I should not be rolling in this change into that. And I see other tests, like StringSetTest using a harness without any members.